src/figuresClasses.ts

```diff
@@ -1,11 +1,81 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const maxSide: number = Math.max(this.a, this.b, this.c);
+
+    if (
+      this.a <= 0 ||
+      this.b <= 0 ||
+      this.c <= 0 ||
+      maxSide >= this.a + this.b + this.c - maxSide
+    ) {
+      throw new Error(
+        'Triangle sides should be positive numbers ' +
+          'and satisfy the triangle inequality',
+      );
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Circle implements Figure {
+  shape: string = 'circle';
 
-export class Circle implements Figure {}
+  constructor(
+    public color: string,
+    public radius: number,
+  ) {
```

Comment:
[CODE KNOWLEDGE] - Similar to the Triangle class, in the Circle class, you don't need to manually declare the 'radius' property on line 45 because TypeScript will automatically create this property from the constructor parameter with an access modifier.